BriefAssistant/ClientApp/app/components/welcome/welcome.component.html

```diff
@@ -0,0 +1,57 @@
+<h2>Welcome to the Brief Assistant!</h2>
+<div>
+    <p>The Brief Assistant <b>WILL</b>:</p>
+    <ul>
+        <li>
+            Show you an example of and Appellant's Initial Brief for the Wisconsin Court of Appeals
+        </li>
+        <li>
+            Explain the different sections of an Appellant's Initial Brief.
+        </li>
+        <li>
+            Direct you to websites where you can find cases, statutes, and other forms of law to cite in your Appellant's Initial Brief.
+        </li>
+        <li>
+            Help you create a draft of a correctly-formatted Appellant's Initial Brief that you can download, save, fill out, edit, and then file in your appeal.
+        </li>
+    </ul>
+</div>
+<div>
+    <p>The Brief Assistant will <b>NOT</b>:</p>
+    <ul>
+        <li>
+            Generate a completed Appellant's Initial Brief for you. You will need to fill out the sections of the Appellant's Initial Brief and complete additional steps before you can file it with the Wisconsin Court of Appeals.
+        </li>
+        <li>
+            Do legal research or tell you what law applies to your case.
+        </li>
+        <li>
+            Read
+            <span>
+                <a href="http://docs.legis.wisconsin.gov/statutes/statutes/809/II/19" target="_blank">Wis. Stat. &sect;809.19</a>
+            </span>
+            for you. Users are responsible for reading and complying with &sect;809.19, which governs the form, contents and filing of appellate briefs.
+        </li>
+        <li>
+            Generate other forms that must be filed in an appeal--such as a Notice of Appeal or a Statement on Transcript.
+        </li>
+    </ul>
+</div>
+<div>
+    <p>
+        <b>Disclaimer:</b>
+    </p>
+    <p>
+        The Brief Assistant does NOT provide legal advice. The Appellate Practice Section strongly urges parties to an appeal to retain a lawyer to represent them. Sometimes that is not possible.
+    </p>
+    <p>
+        So if you have filed a <b>civil</b> appeal and you cannot afford a lawyer, you may wish to call, text, or email the Appellate Help Desk at 414-671-9041 or
```

Comment:
Make the phone number a link too, that way people on mobile wont need to dial the number.
`<a href="tel:+1-414-671-9041">414-671-9041</a>`

Reply:
Added